api/schema/tag_to_label_resource.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Illumio, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package schema
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	resource_schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+var (
+	tagToLabelResource = Resource{
+		TypeName: "tag_to_label",
+		Schema: resource_schema.Schema{
+			Version:             1,
+			MarkdownDescription: "Maps AWS account tags to CloudSecure labels.",
+			Attributes: map[string]resource_schema.Attribute{
+				IDFieldName: idAttribute,
+				"cloud_tags": resource_schema.ListAttribute{
+					MarkdownDescription: "List of tags to map to CloudSecure labels with the specified key. The values of the created labels correspond to the values of the tags. The cloud field for each tag must be \"aws\" or \"azure\" or \"oci\".",
+					Required:            true,
+					ElementType: types.ObjectType{
+						AttrTypes: map[string]attr.Type{
+							"key":   types.StringType,
+							"cloud": types.StringType,
+						},
+					},
+				},
```

Comment:
1. To be consistent with how we're modeling Deployments (#128) and our other resources,
    1. Remove the `cloud` field altogether.
    1. Split this into two fields: `aws_tags`, `azure_tags`.
2. Make these Sets instead of Lists.
```suggestion
				"aws_tag_keys": resource_schema.SetAttribute{
					Description: "Sets of keys of AWS resource tags to map to CloudSecure labels with the same keys. The values of the created labels correspond to the values of the tags.",
					Required:    true,
					ElementType: types.StringType,
				},
				"azure_tag_keys": resource_schema.SetAttribute{
					Description: "Set of keys of Azure resource tags to map to CloudSecure labels with the same keys. The values of the created labels correspond to the values of the tags.",
					Required:    true,
					ElementType: types.StringType,
				},
```